rest-api-spec/src/main/resources/rest-api-spec/test/indices.stats/13_fields.yml

```diff
@@ -71,6 +71,7 @@ setup:
 
   - do:
       search:
+        allow_partial_search_results: true
```

Comment:
Could you explain why this is needed? This test doesn't appear to test partial results?